### DISCUSSION TITLE
Setting auto width and height

### DISCUSSION THREAD

**Discussion by yashkathe** [recharts/recharts#4638] [Q&A] [answered]
How can I set the width and height to be 100% so that I can create a custom component wrapper to handle width and height for different screen sizes

**Comment by ckifer** [comment, marked as answer]
You can use ResponsiveContainer or you can use your own "resize observer" implementation and pass those height and width values down to your chart

**Reply by yashkathe** [reply]
Thank you so much ResponsiveContainer works well !
